components/tabs/tabs.js

```diff
@@ -552,22 +560,9 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 		}
 	}
 
-	_handleFocusEnd(e) {
-		if (e.relatedTarget && e.relatedTarget.role !== 'tab') {
-			this._focusSelected();
-		} else {
-			const nextFocusable = getNextFocusable(e.target, false);
-			if (nextFocusable) nextFocusable.focus();
-		}
-	}
-
-	_handleFocusStart(e) {
-		if (e.relatedTarget && e.relatedTarget.role === 'tab') {
-			const previousFocusable = getPreviousFocusable(e.target, false);
-			if (previousFocusable) previousFocusable.focus();
-		} else {
-			this._focusSelected();
-		}
+	_handleFocusOut(e) {
+		if (!e.relatedTarget || e.relatedTarget.role === 'tab') return;
```

Comment:
If the `focusout` event is being generated because we're moving between tabs, don't reset focusables.

Reply:
That said, I think this check is wrong. I believe it should be:
```e.relatedTarget && e.relatedTarget.role === 'tab'```

If the relatedTarget is `null`, that means we're potentially removing focus from the document entirely, but we still want to reset focusables, I imagine.